app/templates/index.html

```diff
@@ -19,16 +19,16 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Things To Do!</h2>
       <ul class="list-group-item">
         <!-- List the todo items here -->
-          {% for item in data %}
+          {% for item in view_model.items %}
             <li>{{ item.title }} - {{item.status}} 
               <table>
-                {% if item.status == 'Not Started' %}
+                {% if item.status == 'To Do' %}
                   <tr>
-                    <form class="completed" action="/update/{{item.id}}" method="POST"><input type="submit" value="Complete"></form>
+                    <form class="completed" action="/update/{{item.id}}" method="POST"><input type="submit" value="Complete" id="{{item.title}}_complete"></form>
```

Comment:
Having an id like this can cause problems since the title can contain any (user entered) character. The templating engine should HTML escape relevant characters here but you'd still need a bit of care in general. An immediate problem you do have is item.title isn't guaranteed to be unique.

Better to use the item's id instead.